std/assembly/dataview.ts

```diff
@@ -6,9 +6,14 @@ import { E_INDEXOUTOFRANGE, E_INVALIDLENGTH } from "./util/error";
 
 export class DataView {
 
-  private data: ArrayBuffer;
-  private dataStart: usize;
-  private dataLength: i32;
+  @unsafe readonly dataStart: usize;
```

Comment:
You mentioned that DataView should extend ArrayBufferView. Is this possible now?

Reply:
Not sure is it good idea. Currently `ArrayBufferView` and `DataView` structurally identical (have same layout and fields) but `ArrayBufferView` use default constructor which don't need in `DataView`